resources/sass/app.scss

```diff
@@ -14,10 +14,16 @@ html {
 
 body {
     @include body-M;
-    @include body-S;
     height: 100%;
 }
 
+// TODO: This is a temporary fix until issue https://phabricator.wikimedia.org/T326900
+// is marked as resolved.
+.cdx-button .cdx-icon, .cdx-button {
```

Comment:
this is the way we found to make the mixin apply for buttons with and without Icons. This also adds the bold font weight, we might consider just adding it to the mixin itself?

Reply:
I would encourage us to add the weight as an argument in the mixin, yes.

Reply:
The `.cdx-button .cdx-icon` part is causing the icon to be rendered in a different color, any properties assigned to the parent `.cdx-button` will be overwritten here. I suggest getting rid of the `.cdx-button .cdx-icon` part here.